packages/esm-patient-medications-app/src/api/api.ts

```diff
@@ -51,7 +51,7 @@ export async function fetchPatientOrders(
   abortController?: AbortController,
 ) {
   const { data } = await openmrsFetch<PatientMedicationFetchResponse>(
```

Comment:
Oh wait, can't we delete this whole function `fetchPatientOrders` function now?

Reply:
Correct! I've moved the hook `usePatientOrders` hook into the `api.ts` file as well for consistency.